### ISSUE TITLE
As a municipal worker, I want to be able to update accessibility data for the stops in my area.

### ISSUE THREAD

**Issue by culka** [HSLdevcom/jore4#1421]
It should be possible for an outside municipal worker to update the accessibility information of a stop. HSL does not manage the stop infrastructure and relies on the information provieded by the cities, so they should be able to directly update the relevant information.

This will remove possible delays in the data update.
